style.css

```diff
@@ -0,0 +1,137 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font-family: inherit;
+  vertical-align: baseline;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  font-family: sans-serif;
+  padding-top: 0;
+}
+
+header {
+  padding: 0 20px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+nav {
+  font-family: Poppins, sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  letter-spacing: 0.1em;
+  text-align: center;
+  padding: 10px;
+  height: 80px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.mob-menu {
+  margin: 0;
+}
+
+.mob-menu img {
+  height: 35px;
+  width: 35px;
+}
+
+.my-logo {
+  color: hsl(234, 100%, 69%);
+  height: 29px;
+  width: 70px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: center;
+  text-decoration: none;
+  font-size: 18px;
+  line-height: 20px;
+  margin: 0;
+  border: 1px solid #6070ff;
+  border-radius: 2px;
+  font-weight: 700;

```

Comment:
- [OPTIONAL] I suggest removing the border from your logo. remember your design should be similar to [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) 👌